app/src/features/profile-settings/hooks/useUpdateProfilePicture.ts

```diff
@@ -0,0 +1,23 @@
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+
+import { UpdateProfilePicture as UpdateProfilePictureAPI } from "../services/UpdateProfilePicture";
```

Comment:
import with the right name

Reply:
This is the convention we use in importing API Functions to differentiate between the API functions and used inside hooks